istioctl/cmd/add-to-mesh.go

```diff
@@ -0,0 +1,204 @@
+// Copyright 2019 Istio Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/ghodss/yaml"
+	"github.com/spf13/cobra"
+	"go.uber.org/multierr"
+	"k8s.io/client-go/kubernetes"
+
+	"istio.io/istio/istioctl/pkg/util/handlers"
+	istiocmd "istio.io/istio/pilot/cmd"
+	"istio.io/istio/pilot/pkg/kube/inject"
+	"istio.io/pkg/log"
+
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8s_labels "k8s.io/apimachinery/pkg/labels"
+
+	meshconfig "istio.io/api/mesh/v1alpha1"
+)
+
+var (
+	// Function that creates Kubernetes client-go; making it a variable lets us mock client-go
+	interfaceFactory = createInterface
+)
+
+func addToMeshCmd() *cobra.Command {
+	addToMeshCmd := &cobra.Command{
+		Use:     "add-to-mesh",
+		Aliases: []string{"add"},
+		Short:   "Add workloads into Istio service mesh",
```

Comment:
Considering adding ```Example: "istioctl experimental add-to-mesh service productpage"``` to make the usage output easier to read.

Reply:
fixed